include/mmtf/structure_data.hpp

```diff
@@ -163,7 +163,7 @@ struct StructureData {
   std::string                              title;
   std::string                              depositionDate;
   std::string                              releaseDate;
-  std::vector<std::vector<float> >         ncsOperatorList;
+  std::vector<std::vector<float>>          ncsOperatorList;
```

Comment:
This is C++11 syntax. As far as I know this project is still C++03 (but you have my vote for dropping that).

Reply:
I'm guessing you meant to comment this somewhere else?